framework/base/Action.php

```diff
@@ -47,6 +48,10 @@ class Action extends Component
      */
     public $controller;
 
+    /**
+     * @var mixed the result of the action, derived class my set or get action result in the afterRun method
```

Comment:
```suggestion
     * @var mixed the result of the action, derived class may set or get action result in the afterRun method.
     * @since 2.0.41
```